src/game/mod.rs

```diff
@@ -27,17 +27,16 @@ impl EventHandler for State {
         self.player1.update(self.crab.location)?;
         self.player2.update(self.crab.location)?;
         self.collision_check();
-        if !self.assets.bg_sound.playing() {
-            let _ = self.assets.bg_sound.play();
-        }
-
+        /*
```

Comment:
This is also interesting. A lot of 2d engines like love2d have a `setup` hook, where I would start playing background-music right after the sound was loaded. Here we basically start playing in the main update loop and that's why have to check every time if it's playing or not. Do you think it's worth mentioning explicitly?

Reply:
I do think so, yes! I thought this might've been a way that the event loop was set up like